 2007-05-28  Jakub Jelinek  <jakub@redhat.com>

	* sysdeps/unix/sysv/linux/futimes.c (miss_utimensat): Don't define
	if __NR_utimensat is not defined.

===================================================================
RCS file: /cvs/glibc/libc/sysdeps/unix/sysv/linux/futimes.c,v
retrieving revision 1.11
retrieving revision 1.12
diff -u -r1.11 -r1.12
--- libc/sysdeps/unix/sysv/linux/futimes.c	2007/05/10 21:42:45	1.11
+++ libc/sysdeps/unix/sysv/linux/futimes.c	2007/05/28 16:42:52	1.12
@@ -1,5 +1,5 @@
 /* futimes -- change access and modification times of open file.  Linux version.
-   Copyright (C) 2002,2003,2005,2006 Free Software Foundation, Inc.
+   Copyright (C) 2002,2003,2005,2006,2007 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
 
    The GNU C Library is free software; you can redistribute it and/or
@@ -29,7 +29,7 @@
 #include <kernel-features.h>
 
 
-#ifndef __ASSUME_UTIMENSAT
+#if defined __NR_utimensat && !defined __ASSUME_UTIMENSAT
 static int miss_utimensat;
 #endif
 
